<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>actionSimple</class>
 <widget class="QDialog" name="actionSimple">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>simple operations</string>
  </property>
  <widget class="QTextEdit" name="textEdit_1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_3">
   <property name="geometry">
    <rect>
     <x>388</x>
     <y>300</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>130</y>
     <width>122</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>расчитать</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_4">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>300</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>555</x>
     <y>310</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>190</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>характеристика</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
